uber/decorators.py

```diff
@@ -30,6 +30,15 @@ def with_check(*args, **kwargs):
     return with_check
 
 
+def redirect_if_at_con_to_kiosk(func):
+    @wraps(func)
+    def with_check(*args, **kwargs):
+        if c.AT_THE_CON:
+            raise HTTPRedirect(c.KIOSK_REDIRECT_URL)
+        return func(*args, **kwargs)
+    return with_check
+
+
 def check_if_can_reg(func):
```

Comment:
Y'know, the more I think about this, the more it seems like making it a decorator was a bit too over-engineered. Like, we only use this decorator in one place, and if we made it more flexible, all we'd be doing is making two lines of code into one line.